src/modules/finance/Finance.cpp

```diff
@@ -172,4 +172,28 @@ std::string Finance::creditCardExpirationDate()
     const auto expirationDate = Date::futureDate(3);
     return expirationDate.substr(5, 2) + "/" + expirationDate.substr(2, 2);
 }
+
+std::string Finance::creditCardType(const std::string& creditCardNumber)
```

Comment:
You did a good job here, but it was about to return just a random credit card type without passing any parameters, please adjust your implementation.

Reply:
I am confused on how I fix this. Where should I look to fix it?

Reply:
construct a vector with strings: Visa, MasterCard,American Express and Discover and return random element from this vector :) simple as that

Reply:
do I construct the vector in this function or another?

Reply:
You can create it in separate file in data directory

Reply:
As many others data files